addons/my-ocular/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Show ocular.jeffalo.net statuses and post reactions",
-  "description": "Shows ocular statuses and reactions. Uses jeffalo.net to obtain data.",
+  "name": "ocular.jeffalo.net statuses and reactions",
```

Comment:
Can we go with something simple such as "ocular integration?"

Reply:
Legal requirement /shrug
Might add a popup for this maybe

Reply:
Mind explaining "legal requirement?" "Integration" here is already a strong word, which describes that it will do some stuff with it.

Reply:
Dude really ghosted me

Reply:
Dude really ghosted me @apple502j

Reply:
Saw @WorldLanguages reacted. What do you think about "ocular integration?"

Reply:
Since there are no objections or opinions... @mxmou can we change this?

Reply:
I'll change it.